app/controllers/api/v1/worksheet_templates_controller.rb

```diff
@@ -32,8 +32,11 @@ def new
   def create
     @worksheet_template = WorksheetTemplate.new(worksheet_template_params)
     @worksheet_template.user = current_user
-    image = Cloudinary::Uploader.upload(params[:worksheet_template][:photo])
-    @worksheet_template.image_url = image
+    if params[:worksheet_template][:photo]
+      image = Cloudinary::Uploader.upload(params[:worksheet_template][:photo])
+      @worksheet_template.image_url = image
+      # else use default empty whiteboard
```

Comment:
Is this supposed to be commented out?

Reply:
yes, not sure yet if the empty whiteboard will be handled by BE or FE, so leaving a comment to not forget in the future